public/scripts/week-calendar.js

```diff
@@ -562,22 +560,19 @@ function submitTaskForm() {
             setTimeout(location.reload.bind(location), 3000);
```

Comment:
on the live above shouldnt it be populateTasksModal(data.userId, data._id ) ???

Reply:
updated the routes to take user id, returning user ID and task id